### ISSUE TITLE
Issue when using cutoff frequencies greater 2 Hz

### ISSUE THREAD

**Issue by gleichaufjo** [kgram007/Eulerian-Motion-Magnification#5]
I have created a ROS implementation out of your c++ code. I am facing the issue that if I am setting the cutoff frequencies higher than 2 Hz that the magnification does not work and the resulting image turns black. I am not sure what this is caused by. Have you come across this issue before? I am trying to use this code in order to magnify the heart beat.

**Comment by kgram007** [comment]
Yes, I too see an issue with the output magnified images for cutoff frequencies > 2.0 Hz. I am not the original author of the paper and merely tried to implement the work using C++ and OpenCV. But I will still try to look at whats happening and see if I can fix.

**Comment by gleichaufjo** [comment]
Is there any news on this issue? Thank you in advance.
